backend/app/api/endpoints/projects.py

```
@@ -176,6 +177,30 @@ def create_project_conversation_endpoint(
+    current_user: User = Depends(get_current_user),
+    db: Session = Depends(get_db),
+):
+    """Archive all active chats in a project."""
+    try:
+        count = project_service.archive_project_chats(
+            db=db, project_id=project_id, user_id=current_user.id
+        )
+        return {"message": "Project chats archived successfully", "count": count}
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)